### PR TITLE
RavenDB-17999: add schema upgrade tests for CompareExchange And Expiration

### PR DESCRIPTION
**Pull request by garayx**
v5.2 for https://github.com/ravendb/ravendb/pull/13642

### CODE REVIEW

File: test/SlowTests/SlowTests.csproj [hunk 1]
```diff
@@ -197,6 +197,9 @@
     <None Update="SchemaUpgrade\Issues\SystemVersion\after_from14.zip">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SchemaUpgrade\Issues\SystemVersion\CompareExchange_Expiration_RavenData_v5.2_new_from50000.zip">
```
Comment (arekpalinski) [inline test/SlowTests/SlowTests.csproj]: There is no file like that added in this PR

File: test/SlowTests/Cluster/CompareExchangeExpirationTest.cs [hunk 2]
```diff
@@ -480,6 +481,37 @@ public async Task CanSnapshotCompareExchangeWithExpiration()
             }
         }
 
+        [Fact]
+        public void AllCompareExchangeAndExpirationPreserveAfterServerSchemaUpgradeFrom50000()
+        {
+            var folder = NewDataPath(forceCreateDir: true);
+            DoNotReuseServer();
+
+            var zipPath = new PathSetting("SchemaUpgrade/Issues/SystemVersion/CompareExchange_Expiration_RavenData_v5.2_from50000.zip");
```
Comment (ppekrol) [inline test/SlowTests/Cluster/CompareExchangeExpirationTest.cs]: let's put a issue number in the path e.g.
```
SchemaUpgrade/Issues/SystemVersion/RavenDB_17999/CompareExchange_Expiration_RavenData_v5.2_from50000.zip
```

File: test/SlowTests/SlowTests.csproj (hunk 1 above)
Comment (garayx) [reply test/SlowTests/SlowTests.csproj]: removed

File: test/SlowTests/Cluster/CompareExchangeExpirationTest.cs (hunk 2 above)
Comment (garayx) [reply test/SlowTests/Cluster/CompareExchangeExpirationTest.cs]: done

**Pull request merged**